### PR TITLE
Add docs on how to use sutty with distributed press

### PR DESCRIPTION
**Pull request by RangerMauve**
This gives a basic walkthrough of the "happy path" for publishing a post using Sutty's Minima theme.

The goal is to have folks follow links in the guide and look at the screenshots to keep track of where they should be and what they should be looking for.

This is not meant to be extensive, and this does not assume that people will be manually navigating to the panel from Sutty's home page.

Main areas for feedback:
- Fix any spelling errors I'm sure to have added by accident
- Fix up wording / rephrase steps
- Point out steps that are hard to follow or should be added/removed

### CODE REVIEW

**Review by jackyzha0 — APPROVED** [review]
Looks good! Pending @maisutton's PR :)

**Comment by RangerMauve** [comment]
Cool, lets merge this in!
